tests/core/propertyTypes.test.js

```diff
@@ -62,4 +62,19 @@ suite('propertyTypes', function () {
       assert.equal(stringify(this.el), '#hello');
     });
   });
+
+  suite('array', function() {
+    var parse = propertyTypes.array.parse;
+    var stringify = propertyTypes.array.stringify;
+
+    test('parses array', function () {
+      assert.deepEqual(parse(''), []);
+      assert.deepEqual(parse('5 test'), ['5', 'test']);
+    });
+
+    test('stringifies array', function () {
+      assert.equal(stringify([]), '');
+      assert.equal(stringify([5, 10]), '5 10');
```

Comment:
I'm ok with just :facepunch: 

Hm, spaces seem cleaner, but then we wouldn't be able to have spaces in our strings. Not sure. @dmarcos?